keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -962,20 +1173,23 @@ _This section must be completed when targeting alpha to a release._
 
 * **How can this feature be enabled / disabled in a live cluster?**
   - [x] Feature gate (also fill in values in `kep.yaml`)
-    - Feature gate name: InPlacePodVerticalScaling
-    - Components depending on the feature gate: kubelet, kube-apiserver, kube-scheduler
-  - [ ] Other
-    - Describe the mechanism:
-    - Will enabling / disabling the feature require downtime of the control
-      plane? No.
-    - Will enabling / disabling the feature require downtime or reprovisioning
-      of a node? No.
+    - Feature gate name: `InPlacePodVerticalScaling`
+      - Components depending on the feature gate: kubelet, kube-apiserver, kube-scheduler
+    - Feature gate name: `InPlacePodVerticalScalingAllocatedStatus`
+      - Components depending on the feature gate: kubelet, kube-apiserver
+      - Requires `InPlacePodVerticalScaling` be enabled
 
-* **Does enabling the feature change any default behavior?** No
+* **Does enabling the feature change any default behavior?**
+
+  - Kubelet sets several pod status fields: `AllocatedResources`, `Resources`
 
 * **Can the feature be disabled once it has been enabled (i.e. can we roll back
   the enablement)?** Yes
 
+  - Feature can be disabled without issue in the control plane.
+  - Can be disabled on nodes, but if there are any pending resizes container resource configurations
+    may be left in an unknown state.
```

Comment:
```suggestion
  - Can be disabled on nodes, but if there are any pending resizes container resource configurations
    may be left in an unknown state. This can be addressed by draining each node.
```

Reply:
Is this accurate?  I'd like to offer a solution here is we can.

Reply:
Yeah, that's right. I added a slight variation. I also noted that the `InPlacePodVerticalScalingAllocatedStatus` feature can be freely disabled.